src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -138,22 +143,43 @@ public boolean hasActualCostFee() {
     return getActualCostFee().isChargeable();
   }
 
-  public static LostItemPolicy unknown(String id) {
-    return new UnknownLostItemPolicy(id);
+  public boolean shouldChargeFeesWhenAgedToLost() {
+    // i.e. we exclude immediate billings that have nothing to charge
+    return shouldDelayBillingForPatronWhenItemAgedToLost()
+      || setCostFee.isChargeable() || ageToLostProcessingFee.isChargeable();
   }
 
   public boolean canAgeLoanToLost(DateTime loanDueDate) {
-    if (agedToLostAfterOverdueInterval == null
-      || agedToLostAfterOverdueInterval.toMinutes() <= 0) {
+    if (agedToLostAfterOverdueInterval.isEmpty()) {
+      return false;
+    }
+
+    if (actualCostFee.isChargeable()) {
+      // actual cost is not supported now
       return false;
     }
     return agedToLostAfterOverdueInterval.hasPassedSinceDateTillNow(loanDueDate);
   }
 
+  public DateTime calculateDateTimeWhenPatronBilledForAgedToLost(DateTime loanDueDate) {
+    // dueDate + age to lost after interval + patron billed after aged to lost interval
+    return loanDueDate
+      .plus(agedToLostAfterOverdueInterval.timePeriod())
+      .plus(patronBilledAfterAgedToLostInterval.timePeriod());
+  }
+
+  public static LostItemPolicy unknown(String id) {
+    return new UnknownLostItemPolicy(id);
+  }
+
+  private boolean shouldDelayBillingForPatronWhenItemAgedToLost() {
+    return !patronBilledAfterAgedToLostInterval.isEmpty();
+  }
+
   private static class UnknownLostItemPolicy extends LostItemPolicy {
     UnknownLostItemPolicy(String id) {
       super(id, null, noAutomaticallyChargeableFee(), noAutomaticallyChargeableFee(),
-        noActualCostFee(), null, false, false, null);
+        noActualCostFee(), zeroDurationPeriod(), false, false, zeroDurationPeriod(), zeroDurationPeriod(), noAutomaticallyChargeableFee());
```

Comment:
Does this line fit within 100 characters?

Reply:
Nope, thanks. fixed now.